docker-compose.yml

```diff
@@ -0,0 +1,14 @@
+version: '3.8'
+
+services:
+    gpt4:
+      build:
+        context: .
+	dockerfile: Dockerfile
+	image: gpt4free:latest
+      container_name: gpt4
+      ports: 
+        - 127.0.0.1:8501:8501
```

Comment:
i suggest not add ip limit

Reply:
Without it, the service would be exposed to the open Internet.